### PR TITLE
Bump integration-tests to v0.10.1

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/integration-tests/pull/293

### CODE REVIEW

**Comment by fridex** [comment]
/hold

... until available https://quay.io/repository/thoth-station/integration-tests?tab=tags

File: integration-tests/overlays/moc-prod-ocp4-stage/imagestreamtag.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
     - name: latest
       from:
         kind: DockerImage
-        name: quay.io/thoth-station/integration-tests:v0.10.0
+        name: quay.io/thoth-station/integration-tests:v0.10.11
```
Comment (mayaCostantini) [inline integration-tests/overlays/moc-prod-ocp4-stage/imagestreamtag.yaml:11]:
```suggestion
        name: quay.io/thoth-station/integration-tests:v0.10.1
```

**Review by mayaCostantini — APPROVED** [review]
thanks 💯

**Comment by fridex** [comment]
/unhold

The built image is available.

**Pull request merged**